src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/UserAccountStub.java

```diff
@@ -266,8 +267,7 @@ public UserAccountEntity createJudgeUser() {
 
     @Transactional
     public UserAccountEntity createJudgeUser(String identifier) {
-        SecurityGroupEntity securityGroupEntity = securityGroupRepository.findById(-3).get();
-        securityGroupEntity.setGlobalAccess(true);
+        SecurityGroupEntity securityGroupEntity = securityGroupRepository.findById(TEST_JUDGE_GLOBAL_SECURITY_GROUP_ID).get();
```

Comment:
Author note: rather that permanently changing the predefined "Test Judge" to give it global access true, I've added a liquibase changeset that creates a predefined "Test Judge Global" (id -7). Using that here